tests/unit/models/code-list-test.js

```diff
@@ -0,0 +1,25 @@
+import { module, test } from 'qunit';
```

Comment:
Do we want this test?

Reply:
I added it in order to more easily test the hasMany issue that was solved with the polymorphic attribute. I don't know if we should keep it.

Reply:
Ok, then we keep it :), you can merge